kubevirtmng/ui/DiskList.jsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import $ from "jquery"
+
+
+class Volume extends React.Component {
+  constructor() {
+      super();
+
+      this.state = {
+         disks: []
+      }
+
+      this.fetchData = this.fetchData.bind(this);
+    };
+
+   componentWillMount() {
+      this.fetchData();
+   };
+
+   fetchData() {
+        $.ajax({
+            type: "GET",
+            dataType: 'json',
+            url: 'http://localhost:8083/v1/volumes/'+this.props.data.name+"/",
+            success: function(response){
+            console.log(response);
+            this.setState({disks: response.disks})
+            }.bind(this)
+        });
+
+   };
+   render() {
+      return (
+        <tr>
+        <td>
+          <table className="datatable table table-striped table-bordered">
+            <tbody>
+            <tr>
+              <td>Volume:{this.props.data.name}
+                <table className="datatable table table-striped table-bordered">
+                   <tbody>
+                      {this.state.disks.map((disk, i) => <Disk key = {i} data = {disk} />)}
+                   </tbody>
+                </table>
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <form className="form-horizontal">
```

Comment:
Any reason not to use <Form ?

I see react-bootstrap as part of your packages. Might save a lot of boilerplate.